linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -109,13 +109,12 @@ object Infos {
   /** Things from a given class that are reached by one method. */
   final class ReachabilityInfoInClass private[Infos] (
       val className: ClassName,
-      val fieldsRead: List[FieldName],
-      val fieldsWritten: List[FieldName],
-      val staticFieldsRead: List[FieldName],
-      val staticFieldsWritten: List[FieldName],
-      val methodsCalled: List[MethodName],
-      val methodsCalledStatically: List[NamespacedMethodName],
-      val jsNativeMembersUsed: List[MethodName],
+      /* We use a single field for all members to reduce memory consumption:
+       * Typeically: There are very few members reached in a single
```

Comment:
Typo + use comma instead of colon?
```suggestion
      * Typically, there are very few members reached in a single
```